config/god/watcher.rb

```diff
@@ -23,8 +21,13 @@ def pid_file
   def watch(*args)
     self.count += 1
 
+    if ! defined?(AppConfig)
```

Comment:
Style/NegatedIf: Favor `unless` over `if` for negative conditions.<br>Style/SpaceAfterNot: Do not leave space between `!` and its argument.